kiali-server/templates/service.yaml

```diff
@@ -34,11 +34,17 @@ spec:
   {{- end }}
     protocol: TCP
     port: {{ .Values.server.port }}
+    {{- if eq .Values.deployment.service_type "nodePort" }}
```

Comment:
the value to look for should be `NodePort` (capital "N"). As per k8s docs [here](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport), let's maintain the same case as the official k8s docs (even if "nodePort" actually works; let's use what the docs say to use).

Reply:
Fixed. Thanks!